REST Bindings/openapi.yaml

```diff
@@ -4333,6 +4644,802 @@ components:
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      example: "in_transit"
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          anyOf: [
+            enum: [
+                "active",
+                "available",
+                "completeness_inferred",
+                "completeness_verified",
+                "conformant",
+                "container_closed",
+                "container_open",
+                "damaged",
+                "destroyed",
+                "dispensed",
+                "disposed",
+                "encoded",
+                "expired",
+                "in_progress",
+                "in_transit",
+                "inactive",
+                "mismatch_class",
+                "mismatch_instance",
+                "mismatch_quantity",
+                "needs_replacement",
+                "no_pedigree_match",
+                "non_conformant",
+                "non_sellable_other",
+                "partially_dispensed",
+                "recalled",
+                "reserved",
+                "retail_sold",
+                "returned",
+                "sellable_accessible",
+                "sellable_not_accessible",
+                "stolen",
+                "unavailable",
+                "unknown"
+              ],
+            format: uri
+          ]
+
+    EQ_persistentDisposition_unset:
+      name: EQ_persistentDisposition_unset
+      description: "If specified, the result will only include events that (a) have a non-null `persistentDisposition` field; and where (b) the value of the `unset` field within the value of the `persistentDisposition` field matches one of the specified values. - see <a href=\"https://ref.mh1.eu/cbv/Disp\" target=\"_blank\">CBV Disposition</a> for standard values.  Standard values should be expressed as bare words, e.g. `in_transit`, whereas custom values should be expressed as URIs or CURIEs for which the namespace prefix is defined. If this parameter is omitted, events are returned regardless of the value of the `unset` field within `persistentDisposition` field or whether the `persistentDisposition` field exists at all."
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      example: "in_transit"
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          anyOf: [
+            enum: [
```

Comment:
cross reference JSON Schema and inject

Reply:
Agreed - we should populate the enum using the injector tool